events/github.html

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <!-- Standard Meta -->
+  <meta charset="utf-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0">
+
+  <!-- Site Properties -->
+  <title>KOSS-Kharagpur Open Source Society</title>
+  <meta  content='KOSS organizes various technical workshops and seminars in the campus in order to continue the healthy culture of Technology and Open Source.' name='description' />
+  <meta content='KOSS, Kharagpur Open Source Society, IIT Kharagpur, open source, KWOC, Winter of Code, Ubuntu install, Gsoc Seminar, Android, workshop, hackathon, Linux environment, Git Development Workflow, Algorithms and Data Structures' name='keywords'/>
+  <link rel="stylesheet" type="text/css" href="../semantic-ui/semantic.css">
+  <link rel="stylesheet" type="text/css" href="../css/homepage.css">
+  <link rel="stylesheet" type="text/css" href="css/event.css">
+  <link rel="icon" href="assets/images/kosslogo.png">
+  <script src="assets/library/jquery.min.js"></script>
+  <script src="semantic-ui/visibility.js"></script>
+  <script src="semantic-ui/sidebar.js"></script>
+  <script src="semantic-ui/transition.js"></script>
+  <script src="../js/homepage.js"></script>
+  <script src="../header.js"></script>
+</head>
+<body>
+
+<!-- Following Menu -->
+   
+   <div class="ui large top menu hidden fixed">
+    <div class="ui container">
+        <!--The menu items are not yet decided but it is recommended to add the page you are working on in this menu-->
+        <!--and add the 'active' class along with 'item' class on your page name-->
+        <div class="item"><img src="../assets/images/kosslogo.png"></div>
+        <div class="right menu">
+          <script>header('home');</script>
+        </div>
+    </div>
+</div>
+
+
+
+<!-- Sidebar Menu -->
+<div class="ui vertical inverted sidebar menu">
+    <a class="active item">Home</a>
+    <a class="item">Work</a>
+    <a class="item">Company</a>
+    <a class="item">Careers</a>
+    <a class="item">Login</a>
+    <a class="item">Signup</a>
+</div>
+
+<!-- Page Contents -->
+<div class="pusher">
+  <div class="ui inverted vertical masthead center aligned segment">
+
+   
+    <div class="ui text container"><br>
+      <h1 class="ui inverted header" style="font-size: 4rem">
```

Comment:
Too much of unnecessary spaces over here. Let's move the actual content a little above if it is feasible.